### PR TITLE
DEL: Remove unused files

### PR DESCRIPTION
**Pull request by marcosrdz**
DEL: Remove unused files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "clean:ios": "rm -fr node_modules && rm -fr ios/Pods && npm i && cd ios && pod update && cd ..; npm start -- --reset-cache",
     "releasenotes2json": "./scripts/release-notes.sh > release-notes.txt; node -e 'console.log(JSON.stringify(require(\"fs\").readFileSync(\"release-notes.txt\", \"utf8\")));' > release-notes.json",
     "branch2json": "./scripts/current-branch.sh > current-branch.json",
+    "prestart": "watchman watch-del $PWD && watchman watch-project $PWD",
```
Comment (marcosrdz) [inline package.json:53]: When debugging on iOS, watchman keeps track of nonexistent instances  which can lead to confusing situations such as code that isnt available anymore to still run.

**Review by Overtorment — APPROVED** [review]

File: ios/BlueWallet-NoLDK.plist [hunk 2]
```diff
@@ -178,21 +178,14 @@
...
 		<string>FontAwesome5_Brands.ttf</string>
 		<string>FontAwesome5_Regular.ttf</string>
 		<string>FontAwesome5_Solid.ttf</string>
-		<string>Foundation.ttf</string>
 		<string>Ionicons.ttf</string>
-		<string>MaterialCommunityIcons.ttf</string>
 		<string>MaterialIcons.ttf</string>
 		<string>Octicons.ttf</string>
-		<string>SimpleLineIcons.ttf</string>
-		<string>Zocial.ttf</string>
```
Comment (Overtorment) [inline ios/BlueWallet-NoLDK.plist:195]: I can't see, those files physically don't exist in repo?

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: ios/BlueWallet-NoLDK.plist (hunk 2 above)
Comment (marcosrdz) [reply ios/BlueWallet-NoLDK.plist:195]: They get picked up from node_modules during build

**Pull request merged**